### DISCUSSION TITLE
如何将“全部文章”每页展示的15篇改成50篇？

### DISCUSSION THREAD

**Discussion by Jeamorg** [Qexo/Qexo#321] [Q&A]
文章太多，找起来比较麻烦，改为50篇能减少点按次数，要怎么改呢？

![图片](https://user-images.githubusercontent.com/93057377/219535533-deaec681-3b0c-4176-95d2-8b41b0ca3cb7.png)

**Comment by am-abudu** [comment]
目前没有办法。
建议你暂时使用搜索功能

**Comment by Jeamorg** [comment]
> 目前没有办法。
> 建议你暂时使用搜索功能

Qexo 不部署前端可当做一个个人云笔记用，以后打算会对文章管理进行优化吗？

**Reply by am-abudu** [reply]
困难。而且近期内事情比较多，只能做简单的维护
